src/sdl2/video.rs

```diff
@@ -1472,6 +1472,25 @@ impl Window {
             Err(get_error())
         }
     }
+
+    pub fn set_opacity(&mut self, opacity: f32) -> Result<(), String> {
```

Comment:
Can you add a comment that 0.0f is transparent and 1.0f is totally opaque? For the "get" method as well, please.

Reply:
Sure, done!